TakoBoutIt/tacoboutit-frontend
# Update testing post review

#### What's this PR do?
Updates testing to reflect new additions relating to posting a new review. Updates apiCalls review post method to hit full coverage. Adds tests to App. Tests new methods in other components. Fixes issue of notches not being detected.
#### How should this be manually tested?
Switch to the branch and run npm test, make sure all tests are passing. Run npm run coverage should be at 80% coverage.
#### Any background context you want to provide?
Still some gaps in testing, but for the most part we are good to go.
#### What are the relevant tickets?
#74 #75 
#### Screenshots (if appropriate)
![Screen Shot 2020-01-07 at 6 27 39 PM](https://user-images.githubusercontent.com/25031031/71942904-081c4e80-317c-11ea-89c9-cb03c41f1d9b.png)
